build.gradle

```diff
@@ -50,7 +50,10 @@ allprojects {
                 if(file(System.env.ALTERNATIVE_IDE).exists()) {
                     alternativeIdePath = System.env.ALTERNATIVE_IDE
                 } else {
-                    throw new GradleException("ALTERNATIVE_IDE path ${System.env.ALTERNATIVE_IDE} not found")
+                    throw new GradleException("ALTERNATIVE_IDE path not found"
+                      + (System.env.ALTERNATIVE_IDE ==~ /.*[\/\\] *$/
+                        ? " (HINT: remove trailing slash '/')"
+                        : ": ${System.env.ALTERNATIVE_IDE}"))
```

Comment:
Should the env value always be printed?

Reply:
I intentionally chose not, because the "hint" is the most important part. When a slash is present, most likely the "hint" will solve the problem. Else the worst case is that the full message will be shown after removing the slash.